### DISCUSSION TITLE
Support for dual-monitor setup?

### DISCUSSION THREAD

**Discussion by jman-schief** [BlitterStudio/amiberry#1563] [Q&A] [answered]
I am Debian/Trixie and use a Wayland compositor (Sway) without Xwayland emulation. I have a second monitor connected to my laptop.

When I launch amiberry 7.0RC2 setting Display -> Screen Mode -> Fullscreen (or Fullwindow), amiberry opens always on the laptop monitor. If I launch amiberry in Windowed display mode, it opens on the monitor I have focused.

I assume dual-monitor setups are not supported, correct?

thanks

**Comment by midwan** [comment, marked as answer]
That is correct, the current version only uses the first/default monitor for everything.
However, most of the code has already been updated internally to be able to handle more than one monitor with a future update. It will come eventually :)
